### ISSUE TITLE
Error when running a local php container after the first time

### ISSUE THREAD

**Issue by bryangruneberg** [uselagoon/lagoon-images#783]
When using a php container locally, there is an error thrown by entrypoint

```
mv: cannot stat '/usr/local/etc/php/conf.d/00-lagoon-php.ini.tpl': No such file or directory
```

I believe this is caused by a line in https://github.com/uselagoon/lagoon-images/blob/b49a5b6ce1b5347cefb9f63fa5eda453746cc193/images/php-fpm/entrypoints/70-php-config.sh#L22  
```
mv "$PHP_INI_DIR/conf.d/00-lagoon-php.ini.tpl" "$PHP_INI_DIR/conf.d/00-lagoon-php.ini" 
```

Because the file is moved, if the container is run a second time the file is not there to be moved by the entrypoint again. 

Should this not be `cp` rather than `mv`?

./cc @seanhamlin - I believe you were working on the new php templating

**Comment by dalibor-matura** [comment]
@guptahemant and I are looking into the same problem as Hemant got this problem after latest build:
`mv: cannot stat '/usr/local/etc/php/conf.d/00-lagoon-php.ini.tpl': No such file or directory`

It seems that the commit https://github.com/uselagoon/lagoon-images/commit/6b3a30bd52aef7b32cb51998d17438ecd5d5e0f4 with Sean's change might be the cause of the problem.

**Comment by sonnykt** [comment]
Confirming that the issue affects both `23.5.0` and latest (`23.6.0`). The older version `23.4.0` is not affected.

**Comment by bryangruneberg** [comment]
good to know its not just my hackaday setup :D 

I'm 90% sure changing the `mv` to `cp` would fix it - but I'd propose we wait for @seanhamlin to weigh in as he'll know the context of why it's done that way.

In the meantime @sonnykt  and @dalibor-matura - rebuilding the container works for my case as a work-around

**Comment by seanhamlin** [comment]
oof, nice spot. I did not consider the event in which the entry points would need to be run multiple times. Sounds like a `cp` might be in order, and a comment to hopefully not trip the next soul up.

**Comment by AlexSkrypnyk** [comment]
@tobybellwood 
Do we know why CI did not pick this up? Can we add `should restart` step in one of the Leia tests?

**Comment by tobybellwood** [comment]
yeah - we certainly can do - it just never crossed our minds that the entrypoint would re-run on an existing image - in Lagoon/Kubernetes it will only ever run on a fresh pull.

**Comment by AlexSkrypnyk** [comment]
@tobybellwood 
sorry for commenting on the closed issue. Should we expect a patched image to be released or wait for `23.7.0`? thanks

**Comment by tobybellwood** [comment]
Just published 23.7.0 🎉
